Northstar.Client/mod.json

```diff
@@ -3,6 +3,7 @@
 	"Description": "Various ui and client changes to fix bugs and add better support for mods",
 	"Version": "1.12.0",
 	"LoadPriority": 0,
+	"InitScript": "_northstar_client_init.nut",
```

Comment:
this should be cl_northstar_client_init, the _ prefix is generally reserved for server scripts

Reply:
(this is for the filename also idk how to review a filename)

Reply:
Addressed in fd62c04c301514e5ad22e269424b606689d8d2ec